### ISSUE TITLE
usable in existing Stable Diffusion (2.1) code?

### ISSUE THREAD

**Issue by Marcophono2** [VoltaML/voltaML#9]
Hello!

Is it possible to use voltaML to accelerate SD without GUI? Or, alternately to the other voltaML repository, can I use voltaML-fast-stable-diffusion also via command line as it is also possible at AUTOMATIC1111? 

I am very impressed by your performance benchmarks! It simply doubles the inference time of a fully driver optimized 4090. 

Best regards
Marc

**Comment by VoltaML** [comment]
Yes, you can accelerate SD without the webGUI. We will update the readme soon to support it. Also, SD 2.0/2.1 support is not there yet.

**Comment by Marcophono2** [comment]
@VoltaML Great! Not that the 2.1 support is still missing. But that it is generally possible. 😄

**Comment by GeneralAwareness** [comment]
Any info on this or does someone just git pull, and have those exact versions required, and nothing else is needed for the speed increase in SD 2.1?
